### PR TITLE
EIP-8 unittests

### PR DESCRIPTION
**Pull request by chfast**
@halfalicious Can you try enabling these tests on Windows?

### CODE REVIEW

**Comment by halfalicious** [comment]
> 
> 
> @halfalicious Can you try enabling these tests on Windows?

I ran them a little while ago and they hit AVs, do we expect your changes to fix the AVs?

**Comment by halfalicious** [comment]
> 
> 
> > @halfalicious Can you try enabling these tests on Windows?
> 
> I ran them a little while ago and they hit AVs, do we expect your changes to fix the AVs?

Actually scratch that, I built and ran your changes and the eip 8 test passes:

```
[----------] 1 test from eip8
[ RUN      ] eip8.test_discovery_packets
[       OK ] eip8.test_discovery_packets (2 ms)
[----------] 1 test from eip8 (2 ms total)
```

**Comment by chfast** [comment]
> Actually scratch that, I built and ran your changes and the eip 8 test passes:

Most of the tests are still disabled on Windows.

File: test/unittests/libp2p/eip-8.cpp [hunk 1]
```diff
@@ -1,40 +1,21 @@
...
 
 using namespace std;
 using namespace dev;
-using namespace dev::test;
 using namespace dev::p2p;
 
-BOOST_FIXTURE_TEST_SUITE(eip8, TestOutputHelperFixture)
-
-BOOST_AUTO_TEST_CASE(test_discovery_packets)
+TEST(eip8, test_discovery_packets)
```
Comment (gumb0) [inline test/unittests/libp2p/eip-8.cpp]: I think we wanted to use camelCase for test names?

Comment (chfast) [reply test/unittests/libp2p/eip-8.cpp]: Would be good at some point to convert names. And maybe add library prefixes.

**Pull request merged**